hibernate-core/src/main/java/org/hibernate/type/descriptor/java/ImmutableMutabilityPlan.java

```diff
@@ -21,10 +21,11 @@
  * @author Steve Ebersole
  */
 public class ImmutableMutabilityPlan<T> implements MutabilityPlan<T> {
-	public static final ImmutableMutabilityPlan INSTANCE = new ImmutableMutabilityPlan();
+	@SuppressWarnings( "rawtypes" )
+	private static final ImmutableMutabilityPlan INSTANCE = new ImmutableMutabilityPlan();
```

Comment:
Please revert and deprecate for removal instead. This is an SPI, so we'd have to first deprecate the field.